specs/regression.txt

```diff
@@ -424,3 +424,13 @@ __a__
 <p>~~*_**__</p>
 <p><strong>a</strong></p>
 ````````````````````````````````
+
+ISSUE 370
+```````````````````````````````` example
+Paragraph 1
+    
```

Comment:
It looks like this blank line has no spaces on it? Maybe your editor trimmed the line? The generated test looks good, however.

Reply:
Woops, it looks like it was just the review pane that stripped the spaces. My bad!